app-drawer-layout/app-drawer-layout.html

```diff
@@ -217,7 +225,8 @@
           var contentContainer = this.$.contentContainer;
 
           if (this.narrow) {
-            drawer.opened = drawer.persistent = false;
+            drawer.opened = this.openedWhenNarrow;
+            drawer.persistent = false;
```

Comment:
There is an issue with the new change.  For example if you have `openedWhenNarrow` set to true on `app-drawer-layout`, and you run the app on Chrome on Android.  Initially you will see the drawer is opened, which is expected.  Now close the drawer and scroll the document content.  You will see the drawer suddenly pop open.  This is because when document is scrolled, the Chrome address bar will open/close and that triggers a window resize event, which causes the `iron-resize` to fire and lead to `resetLayout` to be called.

Currently we have a few entry points to have `resetLayout` to be called.  I think we should move the code that changes the drawer's state to another function and only call that when `narrow` changes.  For example, this code 
```
drawer.opened = this.openedWhenNarrow;
drawer.persistent = false;
contentContainer.classList.add('narrow');
```
should be moved to another function (e.g. narrowChanged) so we don't have to call this when `app-drawer-layout` receives `iron-resize` event.